content/posts/2019-05-21---Working-In-Ember.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Working In Ember"
+date: "2019-05-27T12:05:03.284Z"
+template: "post"
+draft: false
+slug: "/posts/working-in-ember/"
+category: "Engineering"
+banner: "https://i.imgur.com/bM8ljia.jpg"
+tags:
+  - "Engineering"
+  - "Lifestyle"
+description: "After years of working in Ember, I recently took a job working in other technologies. I thought I'd take a moment to reflect on my time with the framework."
+---
+
+![Laptop and notepad](https://i.imgur.com/bM8ljia.jpg)
+<aside>Photo by Lukas from Pexels</aside>
+
+I worked with Ember.js for several years, across multiple companies.
+In fact, it was the first JavaScript framework I had worked in professionally.
+I am so grateful for my experience with the community and the learning curve -- I am a better developer because of the conventions it forced on me.
+But after 4+ years of writing Ember, I took another job where I won't be writing it anymore, and I have a few thoughts overall on the framework.
```

Comment:
I think the overall of this post is a post of reflection as you are closing a chapter in your life with Ember. Maybe emphasize the point that this is "a few thoughts" and it makes the rest of the post have more context.